README.md

```diff
@@ -14,9 +14,10 @@ It includes:
 - Only Gen 2 Cloud Functions are supported by this module. 
 - All resources must be in the same project. 
 - The Cloud Functions defined in the OpenAPI spec must also be specified in the `cloud_functions` variable so that we can configure IAM permissions for the API Gateway. 
-- The `api_spec_file` variable must be a path to the OpenAPI spec YAML file in the same directory as the Terraform configuration file (see example below).
+- The `api_spec_file` variable must be a path to the OpenAPI spec YAML file in the same directory as the Terraform configuration file (see example YAML below).
 - When specifying the `api_key_restrictions` variable, at least one of `ip_restrictions` or `hostname_restrictions` must be specified (ie. not both). 
-
+- When setting the `title` of the API in the OpenAPI spec, be sure to use a very unique name.  
```

Comment:
Add some more content here about WHY we need the unique name - Module only enables the given id in the project you are operating on - so if two private APIs are named the same, the api gateway request might get routed to the non enabled version from the other project and fail.

Each project, within the same org must have different API Titles in the API Spec to remove this isue